pdns/Makefile.am

```diff
@@ -236,6 +236,11 @@ pdns_server_LDADD = \
 	$(LIBCRYPTO_LIBS) \
 	$(SYSTEMD_LIBS)
 
+if HAVE_LUA_RECORDS
+pdns_server_SOURCES += lua-record.cc minicurl.cc minicurl.hh
```

Comment:
Please add these files to `EXTRA_DIST` as well

Reply:
Done !